test/e2e/handler/nns_update_timestamp_test.go

```diff
@@ -22,30 +18,55 @@ var _ = Describe("[nns] NNS LastSuccessfulUpdateTime", func() {
 		originalNNSs map[string]nmstatev1beta1.NodeNetworkState
 	)
 	BeforeEach(func() {
+		By("Sleep 5 seconds to wait for force refresh applied nncp at other tests")
```

Comment:
Do you think it would make sense to add a function that would wait for nns to not change for 2-3 intervals?
Something similar as they do in k8s here
https://github.com/kubernetes/kubernetes/blob/02b9dd6a6e7d2928945afa4b8aedd7650d6d4cab/test/e2e/apimachinery/resource_quota.go#L301

Reply:
Sure, let's do it

Reply:
I think I am going to do this different, I am going to just compare the NNS update timstamp to be > to the original one, since we are testing the enqueueafter at unit test.